site/content/docs/5.3/components/title-bars.md

```diff
@@ -19,31 +19,31 @@ See them in action in our [examples page]({{<docsref "/examples/title-bars">}}).
 
 ## Background color
 
-To change the appearance of a title bar, use our [background utilities]({{<docsref "/utilities/background">}}). Orange branded background utilities are `.bg-dark`, `.bg-white` and `.bg-supporting-*`.'
+To change the appearance of a title bar, use our [background utilities]({{<docsref "/utilities/background">}}). Orange branded background utilities are `.bg-supporting-*`.'
 
-Title bars can be displayed with black or white backgrounds. When using black or white backgrounds, illustrations can be included as an option, but it's not necessary.
+Title bars can be displayed with dark or white backgrounds. When using dark or white backgrounds, illustrations can be included as an option, but it's not necessary.
```

Comment:
It's rather `black` or `white`, or `.bg-dark` and without classes. Wording to rephrase, it's not that clear in this version.